Gruntfile.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['fh:test','eslint']);
```

Comment:
we used to have different order. eslint first. 
I see that there is unit task bellow with jshint - with that change it will be misleading what we using here.